packages/react/src/AnchoredOverlay/AnchoredOverlay.features.stories.tsx

```diff
@@ -2,17 +2,55 @@ import React, {useEffect, useRef, useState} from 'react'
 import type {Args, Meta} from '@storybook/react'
 import {FocusKeys} from '@primer/behaviors'
 
-import {Box, Button} from '..'
+import {Avatar, Box, Button, Octicon, Text} from '..'
 import {AnchoredOverlay} from '../AnchoredOverlay'
 import Heading from '../Heading'
 import {registerPortalRoot} from '../Portal'
 import {Playground} from './AnchoredOverlay.stories'
+import {LocationIcon, RepoIcon} from '@primer/octicons-react'
 
 export default {
   title: 'Components/AnchoredOverlay/Features',
   component: AnchoredOverlay,
 } as Meta
 
+const hoverCard = (
+  <Box p={3} minWidth={320}>
+    <Box display="flex" flexDirection="column" sx={{position: 'relative'}}>
+      <Avatar src="https://avatars.githubusercontent.com/u/92997159?v=4" size={48} />
+      <Box display="flex">
+        <Text weight="medium">monalisa</Text>
+        <Text color={'var(--fgColor-muted)'} ml={1}>
+          Monalisa Octocat
+        </Text>
+      </Box>
+      <Text fontSize={14}>Former beach cat and champion swimmer. Now your friendly octapus with a normal face.</Text>
+      <Box display="flex" mt={1}>
+        <Octicon color={'var(--fgColor-muted)'} icon={LocationIcon} />
+        <Text fontSize={12} ml={1} color={'var(--fgColor-muted)'}>
+          Interwebs
+        </Text>
+      </Box>
+      <Box display="flex" mt={1}>
+        <Octicon color={'var(--fgColor-muted)'} icon={RepoIcon} />
+        <Text fontSize={12} ml={1} color={'var(--fgColor-muted)'}>
+          Owns this repository
+        </Text>
+      </Box>
+      <Button
+        size="small"
+        sx={{
+          position: 'absolute',
+          right: 0,
+          top: 0,
+        }}
+      >
+        Follow
+      </Button>
+    </Box>
+  </Box>
+)
+
```

Comment:
Open to suggestions if there's a more "primer-way" to do this, I'm not 100% primer-versed yet 😅
<img width="373" alt="image" src="https://github.com/user-attachments/assets/c171ccb5-c7ad-4e5c-897c-0d50852ff8b3">

Reply:
This all seems fine to me, but here are two tiny suggestions:
- Replace `Box`es styled with `display: flex` with [Stack](https://primer.style/components/stack/react/alpha)
- Use a flexbox to lay out the "follow" button next to the avatar instead of `position: absolute`

Reply:
Awesome, thanks!!